R/validationpdf.R

```diff
@@ -160,6 +160,8 @@ validationpdf <- function(x,hist,file="validation.pdf",style="comparison", only_
     x <- sub(" historical", "", x, fixed=TRUE)
     # replace remaining .-separators
     x <- gsub(".", " | ", x, fixed=TRUE)
+    # replace "%" with "percent" to avoid error in tex
+    x <- gsub("%", "percent", x, fixed=TRUE)
```

Comment:
I think the version with backslash is what the user usually wants:
```suggestion
    x <- gsub("%", "\\%", x, fixed = TRUE)
```